users/utils.py

```diff
@@ -0,0 +1,25 @@
+import jwt
+
+from django.http            import JsonResponse
+from django.core.exceptions import ObjectDoesNotExist
+
+from users.models           import User
+from my_settings            import SECRET_KEY
+
+def LoginDecorator(func):
```

Comment:
```suggestion
def login_decorator(func):
```
함수 이름은 소문자 스네이크 케이스로 하는 것이 파이썬의 컨벤션입니다.

Reply:
넵 수정하여 진행하겠습니다!